README.md

```diff
@@ -6,17 +6,17 @@
 
 [Get it on Github](https://github.com/status-im/status-mobile/releases)
 
-Join us in creating a browser, messenger, and gateway to a decentralized world. Status is a free (libre) open source mobile client targeting Android & iOS built entirely on [Ethereum](https://ethereum.org/) technologies. That's right, no middle men and `go-ethereum` running directly on your device.
+Join us in creating a browser, messenger, and gateway to a decentralized world. Status is a free (libre) open source mobile client targeting Android & iOS built entirely on [Ethereum](https://ethereum.org/) technologies. That's right, no middlemen and `go-ethereum` running directly on your device.
 
 [![Getting started with Status](https://i.imgur.com/C0aZZEL.jpg)](https://www.youtube.com/watch?v=oDCSEmS9c3o "Getting started with Status")
 
 ## Why?
 
 We believe in a medium of pure free trade, economies with fair, permission-less access and a world without intermediaries. We want to create policies that can exist between friends or scale globally, we want to communicate securely and be uninhibited by legacy systems.
 
-We want to take responsibility for our data, the way we conduct ourselves privately and promote this way of life to a mass audience.
+We want to take responsibility for our data, and the way we conduct ourselves privately and promote this way of life to a mass audience.
```

Comment:
the change here does not seem necessary but still the `and` does not make it not okay also so i guess its fine